src/atwho.js

```diff
@@ -145,9 +145,9 @@ define([
                             // Gracelessly force focus back to input...otherwise, later code steals
                             // the focus, or, occasionally, gives focus to a different input.
                             var $inputor = this.$inputor;
-                            _.delay(function() {
+                            _.defer(function() {
```

Comment:
no, although it does still need code to handle focus & selection correctly: https://github.com/dimagi/Vellum/pull/671/commits/2c292c49c9f2023318abf7b544becf8c4a3ebb5e